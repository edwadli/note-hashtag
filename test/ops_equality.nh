if 42 == 42 then PrintEndline "42 == 42" else PrintEndline "42 != 42"
be PrintEndline "-42 != 42" unless -42 == 42 inwhichcase PrintEndline "-42 == 42"
if true == true then PrintEndline "world will not end today" else PrintEndline "world may end soon"
if false != true then PrintEndline "all ok" else PrintEndline "felt a disturbance in the force"
if "foo" == "foo" then PrintEndline "strings gonna string" else PrintEndline "strings :("
if "foo" != "Foo" then PrintEndline "string neq OK" else PrintEndline "strings :(("
