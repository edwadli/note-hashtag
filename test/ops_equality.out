42 == 42
-42 != 42
world will not end today
all ok
strings gonna string
string neq OK
